setup.py

```diff
@@ -40,16 +40,16 @@ def _install_requirements():
     return requirements
 
 setup(
-    name='atomicapp',
-    version='0.1.9',
```

Comment:
also, it would make it easier for contributors which version they're working on by simply
```bash
cat VERSION
```
the dir